src/Service/SpalpConfig.php

```diff
@@ -47,4 +47,26 @@ public function getAppIds() {
     return $event->getIds();
   }
 
+  /**
+   * Get json schmea config from the module's JSON file.
+   *
+   * @param string $module
+   *   The machine name of the module.
+   *
+   * @return array
+   *   Array representation of the configuration schema settings.
+   *
+   * @todo Merge this method with getConfigFromJson method present on pull request #7
+   */
+  public function getConfigSchemaJson($module) {
+    $json = [];
+    // Get the JSON file for the module.
+    $filename = DRUPAL_ROOT . '/' . drupal_get_path('module', $module) . "/$module.config.schema.json";
+    if (file_exists($filename)) {
+      $string = file_get_contents($filename);
+      $json = json_decode($string);
```

Comment:
We seem to be using Json::decode elsewhere in this PR, which is better to use?

Reply:
I left same comment here
https://github.com/Capgemini/spalp/pull/14/files#r216594074

We should be using Json::decode
See: https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Component%21Serialization%21Json.php/function/Json%3A%3Adecode/8.2.x